.github/workflows/lint.yml

```
@@ -335,12 +335,11 @@ jobs:
       - name: Run clang-tidy
         run: |
           cd "${GITHUB_WORKSPACE}"
+          # The Docker image has our custom build, so we don't need to install it
           python3 -m tools.linter.clang_tidy \
+            --clang-tidy-exe "$(which clang-tidy)" \
             --diff-file pr.diff \
-            --parallel \
-            --verbose \
-            "$@" >"${GITHUB_WORKSPACE}"/clang-tidy-output.txt
-          cat "${GITHUB_WORKSPACE}"/clang-tidy-output.txt
+            --verbose 2>&1 | tee "${GITHUB_WORKSPACE}"/clang-tidy-output.txt
```

```suggestion
            2>&1 | tee "${GITHUB_WORKSPACE}"/clang-tidy-output.txt
```

do we need `--verbose` in CI? it's nice for us but not very useful for a random developer looking for their failures https://github.com/pytorch/pytorch/pull/61674/checks?check_run_id=3072761257